internal/host/host_test.go

```diff
@@ -945,16 +956,82 @@ var _ = Describe("UpdateInventory", func() {
 		common.DeleteTestDB(db, dbName)
 	})
 
+	It("Test populate eligibility", func() {
```

Comment:
usually when writing table driven tests we are using `It` in each test case in the table, it will make it a separate test.
So i think that this should be a context and each test case should have a name and should be a separate test.